controllers/front/CommentGrade.php

```diff
@@ -29,17 +29,30 @@ class ProductCommentsCommentGradeModuleFrontController extends ModuleFrontContro
 {
     public function display()
     {
-        $idProduct = Tools::getValue('id_product');
+        $idProducts = Tools::getValue('id_products');
         /** @var ProductCommentRepository $productCommentRepository */
+
+        if (!is_array($idProducts)) {
+            die();
```

Comment:
You should never stop an execution like this.

Use return $this->ajaxRender(); or something similar instead

Reply:
Change added